.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 yarn-error.log
 
 coverage/
+.vscode
```

Comment:
IDE files should be added to your local gitignore at https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer